### PR TITLE
Update @patternfly/react-core so that the banner component is available

### PR DESCRIPTION
**Pull request by JPinkney**
### What does this PR do?
This PR updates the version of patternfly so that the [banner component](https://www.patternfly.org/v4/components/banner/) is available. This new version of Patternfly seems to mess up the padding on the page-section so I had to use a custom class instead of the default Patternfly noPadding

CQ: https://dev.eclipse.org/ipzilla/show_bug.cgi?id=22936

### Which issue does this PR related to?
Pre-req for https://github.com/eclipse/che/issues/18490

Signed-off-by: Josh Pinkney <joshpinkney@gmail.com>

### CODE REVIEW

**Comment by olexii4** [comment]
@JPinkney Do you have the needed CQ(@patternfly/react-core@4.84.4)?

**Comment by JPinkney** [comment]
I've just created the CQ and added the link to the description

**Review by olexii4 — APPROVED** [review]
LGTM

File: package.json [hunk 1]
```diff
@@ -39,7 +39,7 @@
   "dependencies": {
     "@eclipse-che/workspace-client": "^0.0.1-1608729566",
     "@patternfly/patternfly": "^4.10.31",
-    "@patternfly/react-core": "^4.18.5",
+    "@patternfly/react-core": "^4.23.1",
```
Comment (akurinnoy) [inline package.json]: Actually, resolved version is `4.84.4`, so let it be put in product.json along with `~` to allow updates for patch releases. 

```suggestion
    "@patternfly/react-core": "~4.84.4",
```

File: src/Layout/ErrorReporter/index.tsx [hunk 2]
```diff
@@ -26,8 +26,7 @@ export class ErrorReporter extends React.PureComponent<Props> {
       <Page>
         <PageSection
           isFilled={true}
-          padding={{ default: 'noPadding' }}
-          className={styles.backdropBackground}
+          className={styles.backdropBackground + ' pf-c-page-section-no-padding'}
```
Comment (akurinnoy) [inline src/Layout/ErrorReporter/index.tsx]: Property `padding={{ default: 'noPadding' }}` works seamlessly for me. Did you have any issues with that?

Comment (JPinkney) [reply src/Layout/ErrorReporter/index.tsx]: No issues with it, it just slightly changes the way the UI looks. If that's fine then I can revert the ErrorReporter change

Comment (akurinnoy) [reply src/Layout/ErrorReporter/index.tsx]: Hmm, I don't see any changes in UI between using property and CSS class. So, I'd like to revert changes in this component.

**Review by akurinnoy — APPROVED** [review]

**Pull request closed without merging**